main.go

```diff
@@ -114,16 +116,18 @@ func main() {
 
 func getConfig() (*Config, error) {
 	cfg, err := ucfgYAML.NewConfigWithFile(configPath)
+	if os.IsNotExist(err) {
+		return &defaultConfig, nil
```

Comment:
Could you add a log message here that the default configs are used as no config was found?

Reply:
Fixed